lucille-core/src/main/java/com/kmwllc/lucille/core/JsonDocument.java

```diff
@@ -47,7 +48,7 @@ public JsonDocument(Document document) throws DocumentException {
 
   /**
    * Creates a new {@link JsonDocument} from a {@link ObjectNode} of key/value pairs. Note: does
-   * not defensively copy the given ObjectNode, so it must not be modified after creation.
+   * not defensively copy the given ObjectNode, so it must not be
```

Comment:
I think the end of this comment was not supposed to be removed?

Reply:
added back in [477bae0](https://github.com/kmwtechnology/lucille/pull/131/commits/477bae0f79f5225089df183d4e4269b9f7c32545)